bcda/models/service.go

```diff
@@ -52,14 +65,119 @@ type service struct {
 
 	cutoffDuration time.Duration
 	sp             suppressionParameters
+	rp             runoutParameters
 }
 
 type suppressionParameters struct {
 	includeSuppressedBeneficiaries bool
 	lookbackDays                   int
 }
 
-func (s *service) GetNewAndExistingBeneficiaries(cmsID string, since time.Time) (newBeneficiaries, beneficiaries []*CCLFBeneficiary, err error) {
+type runoutParameters struct {
```

Comment:
We'll likely use these same parameters for other scenarios (i.e., termination).  Should we generalize the name?  Or is it appropriate to keep it as `runoutParameters` for now?

Reply:
I think it's appropriate to keep the name as is. It's internal configuration parameters based on env vars. It won't be generated based on user requests.

Thinking the termination would be driven by user parameters (i.e. terminiation date) and would be supplied to the service.